sys-utils/renice.c

```diff
@@ -59,16 +59,22 @@ static void __attribute__((__noreturn__)) usage(void)
 	FILE *out = stdout;
 	fputs(USAGE_HEADER, out);
 	fprintf(out,
-	      _(" %1$s [-n] <priority> [-p|--pid] <pid>...\n"
-		" %1$s [-n] <priority>  -g|--pgrp <pgid>...\n"
-		" %1$s [-n] <priority>  -u|--user <user>...\n"),
+	      _(" %1$s [-n|--priority|--relative] <priority> [-p|--pid] <pid>...\n"
+		" %1$s [-n|--priority|--relative] <priority>  -g|--pgrp <pgid>...\n"
+		" %1$s [-n|--priority|--relative] <priority>  -u|--user <user>...\n"),
 		program_invocation_short_name);
 
 	fputs(USAGE_SEPARATOR, out);
 	fputs(_("Alter the priority of running processes.\n"), out);
 
 	fputs(USAGE_OPTIONS, out);
-	fputs(_(" -n, --priority <num>   specify the nice value\n"), out);
+	fputs(_(" -n <num>               specify the nice value\n"), out);
+	fputs(_("                          If POSIXLY_CORRECT flag is set in environment\n"), out);
+	fputs(_("                          environment then the priority is 'relative'\n"), out);
```

Comment:
It seems like a typo, `in environment environment then`

Reply:
prio is const, so I updated the code to use newprio instead, but otherwise you were right and it is now corrected.